docs/resources/samples/external-calls.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Make external API calls in Office Scripts'
+description: 'Learn how to make external API calls in Office Scripts.'
+ms.date: 03/18/2021
+localization_priority: Normal
+---
+
+# External API calls from Office Scripts
```

Comment:
I think we need to merge this information with:
- This article: https://docs.microsoft.com/en-us/office/dev/scripts/develop/external-calls
- And this similar sample: https://docs.microsoft.com/en-us/office/dev/scripts/resources/scenarios/noaa-data-fetch

Reply:
Tracking with [feature](https://office.visualstudio.com/OC/_workitems/edit/4924924)